dev-python/cloudscraper/cloudscraper-1.2.65.ebuild

```diff
@@ -31,6 +31,8 @@ BDEPEND="
 	)
 "
 
+RESTRICT=test
```

Comment:
`RESTRICT="test"` I'd also suggest to leave a comment about why are you restricting it, just to be sure if in a later day someone (ir you) want to take a look into it will find your comment about it